observability/akka-http-metrics/src/test/suite/scala/com/daml/metrics/akkahttp/OpenTelemetryUtils.scala

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics.akkahttp
+
+import scala.annotation.tailrec
+
+import com.daml.metrics.api.MetricHandle
+import com.daml.metrics.api.opentelemetry.OpenTelemetryFactory
+import io.opentelemetry.api.metrics.{Meter => OtelMeter}
+import io.opentelemetry.sdk.metrics.SdkMeterProvider
+import io.opentelemetry.sdk.metrics.data.MetricData
+import io.opentelemetry.sdk.testing.exporter.InMemoryMetricReader
+import io.opentelemetry.sdk.testing.time.TestClock
+import java.time.Duration
+import java.util.concurrent.atomic.AtomicInteger
+
+/** Data structure used to compare the content of histogram metrics
+  * The exact recorded values are not accessible, using the other attributes to check that
+  * the data was correctly recorded.
+  */
+case class HistogramData(
```

Comment:
To make it easier to find these classes and to align with coding guidelines, I'd suggest moving `HistogramData` into a separate source file and renaming this file to `TestMetricsBase`.

Reply:
This will go away when we switch to the more generic testing framework from Nicu PR.
And also, where are the coding guidelines ? I cannot find it referenced in the READMEs.